api/src/identity-access-management/domain/models/AuthenticationMethod.js

```diff
@@ -99,11 +101,20 @@ class AuthenticationMethod {
     this.createdAt = createdAt;
     this.updatedAt = updatedAt;
     this.userId = userId;
+    this.lastLoggedAt = lastLoggedAt ? lastLoggedAt : null;
```

Comment:
Il n'y a pas besoin de mettre ce traitement, une simple assignation fait l'affaire : 
```suggestion
    this.lastLoggedAt = lastLoggedAt;
```

En ce qui concerne la validation, `lastLoggedAt: Joi.date().allow(null).optional()` autorise la valeur `undefined` : 
> any.optional()
> Marks a key as optional which will allow undefined as values. Used to annotate the schema for readability as all keys are optional by default.

cf. https://joi.dev/api/?v=17.13.3#anyoptional